lib/bundler/version.rb

```diff
@@ -7,7 +7,7 @@ module Bundler
   # We're doing this because we might write tests that deal
   # with other versions of bundler and we are unsure how to
   # handle this better.
-  VERSION = "2.0.0.dev" unless defined?(::Bundler::VERSION)
+  VERSION = "2.1.0.beta1" unless defined?(::Bundler::VERSION)
```

Comment:
Should I change this to `.dev`? I think we should set the real case like `.beta1`.

Reply:
we don't use `beta` in releases. It would `2.1.0.pre.1`

Reply:
:+1:

I don't have a strong opinion, but I slightly prefer `.beta1`, yeah.

Given how rubygems version comparison works, `2.1.0.dev` ranks higher than `2.1.0.beta1`, so it could potentially lead to some confusion. I prefer to set this to the minimum target version that we might release next. If we end up deciding to go straight to `rc1` for example, we can change it again when we make that decision.

Reply:
> we don't use beta in releases. It would 2.1.0.pre.1

That's true. Let's go with `2.1.0.pre.1` then :+1:.